### DISCUSSION TITLE
connect paths

### DISCUSSION THREAD

**Discussion by tomasweiss** [OpenOrienteering/mapper#2075] [Questions & Answers]
hi , is there a good way to connect paths as required by new mtbo spec?
![Screenshot_20220628-100122_Drive](https://user-images.githubusercontent.com/98590051/176114685-2b55f875-83fc-436a-84eb-b6d7c11ca87b.jpg)

**Comment by dg0yt** [comment]
This is not a new thing, but also required for regular maps. The relevant concept is called "dash point", and it marks the middle of a dash.  You can toggle the dash point property of any control point by `Space +  Click`. If you need to add a control point first, select the object and do `Ctrl + Click`.

What we don't have in Mapper now is true "junction points" as in electronics circuit design.

**Comment by tomasweiss** [comment]
great, thanks!!!
